src/libraries/System.Private.CoreLib/src/System/Runtime/Loader/LibraryNameVariation.Windows.cs

```diff
@@ -10,13 +10,14 @@ internal partial struct LibraryNameVariation
     {
         private const string LibraryNameSuffix = ".dll";
 
-        internal static IEnumerable<LibraryNameVariation> DetermineLibraryNameVariations(string libName, bool isRelativePath)
+        internal static IEnumerable<LibraryNameVariation> DetermineLibraryNameVariations(string libName, bool isRelativePath, bool forSystemApi = false)
```

Comment:
You may want to add the same argument to the `.Unix.cs` file, so that the callsite does not need platform-specific ifdef.